MAUI/TreeMap/legend.md

```diff
@@ -617,6 +617,147 @@ N> This property will only be applicable when the [ShowLegend](https://help.sync
 
  ![legend-size-in-maui-tree-map](images/legend/legend-size-in-maui-tree-map.png)
 
+
+## Items Layout 
```

Comment:
Items Layout - Change as Customize items layout

Reply:
updated the changes